### ISSUE TITLE
Data Files: Timestamp select elements do not have accessible names

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/DSS#575]
### Steps to recreate
Visit /data_files/new or /data_files/_<data_file.id>_/edit and run the Axe dev tools

### URL
http://localhost:3000/data_files/new
http://&#8203;localhost:3000/data_files/_<data_file.id>_/edit

### Issues
Select elements for the Timestamp field of new study creation do not have accessible names

### Severity
Critical

### WCAG Success Criterion
- [ ] Select element must have an accessible name (5)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
When accessible names for select elements are absent, screen reader users do not know the input data expectations.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:
- Element does not have an implicit (wrapped) <label>
- Element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved

### Notes
Applies for all 5 select elements: year, month, day, hour, minute
